.github/CODEOWNERS

```diff
@@ -239,6 +239,7 @@ packages/react-components/react-jsx-runtime @microsoft/teams-prg
 packages/react-components/react-toast @microsoft/teams-prg
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28133&commitId=1ca4737a80af32dfeb32eb7cb62910c667b2dfea&env=prod&clientType=FLUENTUI)


  <details><summary><b>Dialog</b> 46 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.alert - Dark Mode.chromium.png|810745| Changed|
|Dialog.Fluid Actions End.chromium.png|913342| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|811105| Changed|
|Dialog.alert - RTL.chromium.png|914231| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|834354| Changed|
|Dialog.alert - High Contrast.chromium.png|833966| Changed|
|Dialog.default - High Contrast.chromium.png|833966| Changed|
|Dialog.actions position start - High Contrast.chromium.png|833967| Changed|
|Dialog.actions position start - RTL.chromium.png|914231| Changed|
|Dialog.actions position start & position end.chromium.png|913900| Changed|
|Dialog.Fluid Actions Start.chromium.png|913342| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|810745| Changed|
|Dialog.default - Dark Mode.chromium.png|810745| Changed|
|Dialog.alert.chromium.png|914232| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|913899| Changed|
|Dialog.actions position start.chromium.png|914232| Changed|
|Dialog.default - RTL.chromium.png|914231| Changed|
|Dialog.default.chromium.png|914232| Changed|
|Dialog.nested - Dark Mode.chromium.png|814922| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|844556| Changed|
|Dialog.nested - RTL.chromium.png|908136| Changed|
|Dialog.nested.chromium.png|908135| Changed|
|Dialog.nested - High Contrast.chromium.png|836245| Changed|
|Dialog.no actions - RTL.chromium.png|907159| Changed|
|Dialog.no title & no actions.chromium.png|901067| Changed|
|Dialog.no actions.chromium.png|907160| Changed|
|Dialog.no title & no actions - RTL.chromium.png|901066| Changed|
|Dialog.no actions - High Contrast.chromium.png|839307| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|828282| Changed|
|Dialog.no actions - Dark Mode.chromium.png|820910| Changed|
|Dialog.no title - High Contrast.chromium.png|839214| Changed|
|Dialog.no title - Dark Mode.chromium.png|818116| Changed|
|Dialog.no title - RTL.chromium.png|908138| Changed|
|Dialog.no title.chromium.png|908139| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|1467617| Changed|
|Dialog.non-modal.chromium.png|1598016| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|666274| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|820134| Changed|
|Dialog.non-modal - High Contrast.chromium.png|1492354| Changed|
|Dialog.non-modal - RTL.chromium.png|1598014| Changed|
|Dialog.title custom action - High Contrast.chromium.png|838714| Changed|
|Dialog.scroll long content - RTL.chromium.png|1053157| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|705105| Changed|
|Dialog.scroll long content.chromium.png|1053207| Changed|
|Dialog.title custom action.chromium.png|907758| Changed|
|Dialog.title custom action - RTL.chromium.png|907758| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>